habitat/plan.sh

```diff
@@ -0,0 +1,46 @@
+
+pkg_name=node
+pkg_origin=nodejs
+pkg_version=6.11.3
```

Comment:
This isn't correct for `master`.

Reply:
Hmm yeah is this probably a typo or something. @elliott-davis you should strip this out. You've included the appropriate functions for determining version from master in this:

```
+pkg_version() {
+  "$(pkg_path_for core/python2)/bin/python2" ../tools/getnodeversion.py
+}
+
+do_before() {
+  update_pkg_version
+}
```